Tests/fixtures/clienthints.yml

```diff
@@ -298,3 +298,55 @@
     model: Fire HD 8 (2018)
   os_family: Android
   browser_family: Chrome
+-
+  user_agent: Mozilla/5.0 (Linux; Android 11; sohKwhigbQ; U; en) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.101 Mobile AvastSecureBrowser/6.6.0 Build/3850 Safari/537.36
+  headers:
+    Sec-CH-UA: '"AvastSecureBrowser";v="6.6.0", " Not A;Brand";v="99.0.0.0", "Chromium";v="98.0.4758.101"'
+    Sec-CH-UA-Platform: "Android"
+    Sec-CH-UA-Mobile: "?1"
+    Sec-CH-UA-Full-Version: "6.6.0"
+    Sec-CH-UA-Platform-Version: "11"
+    Sec-CH-UA-Arch: "[arm64-v8a, armeabi-v7a, armeabi]"
+    Sec-CH-Prefers-Color-Scheme: "light"
+  os:
+    name: Android
+    version: "11"
+    platform: "ARM"
+  client:
+    type: browser
+    name: Avast Secure Browser
+    version: "6.6.0"
+    engine: WebKit
+    engine_version: "537.36"
+  device:
+    type: tablet
+    brand: ""
+    model: ""
+  os_family: Android
+  browser_family: Chrome
+-
+  user_agent: ""
+  headers:
+    Sec-CH-UA: '"AvastSecureBrowser";v="6.6.0", " Not A;Brand";v="99.0.0.0", "Chromium";v="98.0.4758.101"'
+    Sec-CH-UA-Platform: "Android"
+    Sec-CH-UA-Mobile: "?1"
+    Sec-CH-UA-Full-Version: "6.6.0"
+    Sec-CH-UA-Platform-Version: "11"
+    Sec-CH-UA-Arch: "[arm64-v8a, armeabi-v7a, armeabi]"
+    Sec-CH-Prefers-Color-Scheme: "light"
+  os:
+    name: Android
+    version: "11"
+    platform: "ARM"
+  client:
+    type: browser
+    name: Avast Secure Browser
+    version: "6.6.0"
+    engine: ""
+    engine_version: ""
+  device:
+    type: ""
+    brand: ""
+    model: ""
+  os_family: Android
+  browser_family: Chrome
```

Comment:
This test is a duplicate of the one above.

Reply:
No. It lacks the useragent. Added that to ensure not everything comes from the useragent fallback, which was the case due to some incorrect code before...